src/main/java/org/kairosdb/core/properties/PropertiesValidator.java

```diff
@@ -0,0 +1,54 @@
+package org.kairosdb.core.properties;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.kairosdb.core.exception.KairosDBException;
+
+/**
+ * Ensure that nothing in the kairosdb.properties file would result in
+ * unexpected behavior. Throw a KairosDBException upon startup if so.
+ */
+
+public class PropertiesValidator {
+    // Field DI is not ideal, but you can't provide defaults with constructor DI.
+    @Inject(optional = true)
+    @Named(Props.ALIGN_DATAPOINT_TTL_WITH_TIMESTAMP)
+    private boolean m_alignDatapointTtlWithTimestamp = false;
```

Comment:
This default value literal `false` is also used in `CassandraConfiguration.java`. Should that file expose a public static final boolean for the default value so both places would use it?